docs/src/main/asciidoc/command-mode-reference.adoc

```diff
@@ -55,22 +56,28 @@ import io.quarkus.runtime.Quarkus;
 import io.quarkus.runtime.annotations.QuarkusMain;
 
 @QuarkusMain
-public class JavaMain {
+public class HelloWorldMain2 implements QuarkusApplication {
```

Comment:
I think I would rather use the same name as before `HelloWorldMain` to avoid having people creating both in their project when experimenting.

Reply:
```suggestion
public class HelloWorldMain implements QuarkusApplication {
```

Reply:
good point, I'll amend it tomorrow and force push to my fork

Reply:
Done